pkg/scheduler/api/resource_info.go

```diff
@@ -75,6 +75,8 @@ func NewResource(rl v1.ResourceList) *Resource {
 			r.Memory += float64(rQuant.Value())
 		case v1.ResourcePods:
 			r.MaxTaskNum += int(rQuant.Value())
+		case v1.ResourceName("count/pods"):
```

Comment:
what about some other fields contains `/` ?

Reply:
other fields are all ScalarResource..

Reply:
l mean it's strange to filter `count/pods` only, what about `count/deployments.apps` and sth like that

Reply:
+1, 

AFAIK, `ResourceQuota` already support CRD, how to handle such case in Volcano?

Reply:
> BTW, `ResourceQuota` also support PriorityClass, how to handle that?

you mean the quota scope?

Reply:
create a new issue to track quota scope?